src/System.Slices/System/ReadOnlySpan.cs

```diff
@@ -333,5 +343,9 @@ public override bool Equals(object obj)
             }
             return false;
         }
+
+        public bool Equals(T[] other) => Equals(new ReadOnlySpan<T>(other));
```

Comment:
Are these necessary when there are implicit conversions in place? Not sure on overload resolution off the top of my head :stuck_out_tongue: